# A basic message for the pose (simplified for 2D)
# Estimates the pose of the robot's origin

float64 x           # x position in meters (inf if N/A)
float64 y           # y position in meters (inf if N/A)
float64 theta       # theta orientation in radians (inf if N/A)

